operator/pkg/controllers/controller.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/awslabs/kit/operator/pkg/apis/infrastructure/v1alpha1"
 	"github.com/awslabs/kit/operator/pkg/errors"
-	"github.com/awslabs/kit/operator/pkg/status"
+	"github.com/awslabs/kit/operator/pkg/result"
```

Comment:
I'd suggest renaming this pkg to `results` maybe? It's that that `result` is such a common word to actually use (especially in Go), even if you don't have actual naming conflicts, every time someone sees `result` they're going to thing it's a local variable (containing a result).

Reply:
Thanks, yeah I will address this as part of the next PR I am working on.